nifi-nar-bundles/nifi-amqp-bundle/nifi-amqp-processors/src/main/java/org/apache/nifi/amqp/processors/AbstractAMQPProcessor.java

```diff
@@ -191,6 +201,14 @@ private Connection createConnection(ProcessContext context) {
         if (vHost != null) {
             cf.setVirtualHost(vHost);
         }
+        // handles TLS/SSL aspects
+        final SSLContextService sslService = context.getProperty(SSL_CONTEXT_SERVICE).asControllerService(SSLContextService.class);
+        final SSLContext sslContext = sslService == null ? null : sslService.createSSLContext();
```

Comment:
Pierre
I was working on merging it but realized that the above no longer compiles. Could you take a look and update?